docs/tempostack/object_storage.md

```diff
@@ -13,7 +13,40 @@ weight: 100
 toc: true
 ---
 
-Tempo Operator supports [Minio](https://min.io/) for TempoStack object storage.
+Tempo Operator supports [AWS S3](https://aws.amazon.com/), [Azure](https://azure.microsoft.com), [Minio](https://min.io/) and [OpenShift Data Foundation](https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation) for TempoStack object storage.
+
+## AWS S3
+
+### Requirements
+
+* Create a [bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html) on AWS.
+
+### Installation
+
+* Deploy the Tempo Operator to your cluster.
+
+* Create an Object Storage secret with keys as follows:
+
+    ```console
+    kubectl create secret generic tempostack-dev-s3 \
+      --from-literal=bucketnames="<BUCKET_NAME>" \
```

Comment:
Where is the `bucketnames` field of the secret used? I couldn't find it in the latest main branch or the open PRs

Reply:
You're right. I copied it from Loki doc and forget about changing this. Thanks!